src/handle_action.cpp

```diff
@@ -955,16 +943,38 @@ static void sleep()
         }
     }
 
-    // ask for deactivation
+    uilist as_m;
+    as_m.text = _( "<color_white>Are you sure you want to sleep?</color>" );
     std::stringstream data;
     if( !active.empty() ) {
+    // (Y)es/(S)ave before sleeping/(N)o
+        as_m.entries.emplace_back( 2, true,
+                                   get_option<bool>( "FORCE_CAPITAL_YN" ) ? 'N' : 'n',
+                                   _( "No." ) );
+        as_m.entries.emplace_back( 1, g->get_moves_since_last_save(),
+                                   get_option<bool>( "FORCE_CAPITAL_YN" ) ? 'S' : 's',
+                                   _( "Yes, and save game before sleeping." ) );
+        as_m.entries.emplace_back( 0, true,
+                                   get_option<bool>( "FORCE_CAPITAL_YN" ) ? 'Y' : 'y',
+                                   _( "Yes." ) );
+        // ask for deactivation
         data << as_m.text << std::endl;
-        data << _( "You may want to turn off:" ) << std::endl;
+        data << _( "You may want to take care of:" ) << std::endl;
         data << " " << std::endl;
         for( auto &a : active ) {
             data << "<color_red>" << a << "</color>" << std::endl;
         }
         as_m.text = data.str();
+    } else {
```

Comment:
```suggestion
    } else {
        // (Y)es/(S)ave before sleeping/(N)o
```

Reply:
For the first time I am doing a review, I did not know how to add a line without deleting))